packages/autorest.typescript/test/smoke/generated/compute-resource-manager/package.json

```diff
@@ -35,7 +35,8 @@
     "@azure/identity": "^2.0.1",
     "@azure-tools/test-recorder": "^3.0.0",
     "@azure-tools/test-credential": "^1.0.0",
-    "mocha": "^7.1.1",
+    "mocha": "^10.0.0",
+    "esm": "^3.2.18",
```

Comment:
just curious, why not add @types/mocha here ?

Reply:
Explained above. I can add it

Reply:
fixed